### PR TITLE
Soul core meaningful

### PR DESCRIPTION
**Pull request by Hornsoup**
Makes for more meaningful Soul core by keeping up to 9% aptitude on ascension. <strike>Predicated on the aptitude fix.</strike>

### CODE REVIEW

File: src/app/game-state/character.ts [hunk 1]
```diff
@@ -374,8 +374,8 @@ export class Character {
       // increase by log2 of whatever is over the softcap
```
Comment (immortalityidle) [inline src/app/game-state/character.ts:374]: Can you update this comment? I'm not good enough at math to get what this is doing right away.

Comment (Hornsoup) [reply src/app/game-state/character.ts:373]: Oop, yeah that's part of the aptitude fix, forgot the comment there. Also added a local variable for jittering the balance as needed.

**Comment by Hornsoup** [comment]
This will likely need updating after moving this code to a method but it's been de-branched.

**Review by ChandlerFerry — APPROVED** [review]
Code looks good, but balance is not something I can weigh in on.

**Review by immortalityidle — APPROVED** [review]

**Pull request merged**